test/Dockerfile

```
@@ -8,7 +8,7 @@ RUN <<EOF
 set -eu
 apt-get update && apt-get install -y --no-install-recommends curl ca-certificates git
 curl -L https://foundry.paradigm.xyz | bash
-~/.foundry/bin/foundryup --install stable
+~/.foundry/bin/foundryup --install v1.4.0
```

We should install Foundry 1.4.3 to match [the version used in the CI](https://github.com/cartesi/dave/blob/1f1249a9d9cadae5ed3b250485ff0c9b6a31f2c6/.github/actions/setup-tools/action.yml#L15).

```suggestion
~/.foundry/bin/foundryup --install v1.4.3
```